lib/perl/Genome/Model/Build/Command/Start.pm

```diff
@@ -143,6 +143,9 @@ sub create_and_start_build {
 
     my $outer_transaction = UR::Context::Transaction->begin();
     $model->build_requested(0);
+    my $anp = $model->analysis_project;
+    my $config_dir = $anp? $anp->environment_config_dir : undef;
```

Comment:
Without additional PRs or changes to our test suite we couldn't meet this requirement with test builds.  Many of the recent builds without an AnP are created_by apipe-tester:
$ genome model build list --filter 'created_at>=2015-11-01,model.analysis_projects.id=""' --nohead -sh created_by | sort | uniq -c

With some improved documentation describing how to assign a SomVar, ClinSeq or random model to an AnP manually and an appropriate amount of notice before the requirement is released to a live snapshot... yes we should require an AnP for all builds to start.   However, is it worth coming back to this in another PR?